gr-utils/modtool/tools/parser_cc_block.py

```diff
@@ -21,6 +21,21 @@ def dummy_translator(the_type, default_v=None):
     return the_type
 
 
+def remove_comments(inp: str) -> str:
+    def comment_removal(match: re.match) -> str:
```

Comment:
Minor, but how about dropping this code back into the outer function. The subfunc doesn't serve any purpose any more and makes it harder to read.

Reply:
not quite sure what you mean here - the callable needs to see `inp`, so it needs to be local

Reply:
Made it simpler; no need for the function aside from really pathological corner cases, so omit.